theme.go

```diff
@@ -58,6 +58,7 @@ type FieldStyles struct {
 // TextInputStyles are the styles for text inputs.
 type TextInputStyles struct {
 	Cursor      lipgloss.Style
+	CursorText  lipgloss.Style
```

Comment:
Do we need to add default styles or does this work with the default?

Reply:
Same as `Cursor`, work as-is :)